satpy/etc/enhancements/abi.yaml

```diff
@@ -32,6 +32,18 @@ enhancements:
           min_stretch: [-26.2, -43.2, 243.9]
           max_stretch: [0.6, 6.7, 208.5]
 
+
+  true_color_with_night_fires:
+    standard_name: true_color_with_night_fires
+    sensor: abi
+    operations:
+      - name: linear_stretch
+        method: !!python/name:satpy.enhancements.stretch
+      - name: sqrt
+        method: !!python/name:satpy.enhancements.gamma
+        kwargs: { gamma: 2.0 }
```

Comment:
I would've expected the enhancement to be `crude` stretch from `0.0` to `1.0` as the individual composites are already stretched to their pre-defined values.

Reply:
I second that, what do you say @simonrp84 ?

Reply:
Sorry, overworked right now - will report back on this as soon as I can but I suspect you're right.